### ISSUE TITLE
Exception: src/TopicModel/DTModel.hpp (436): No document with timepoint = 0

### ISSUE THREAD

**Issue by ankane** [bab2min/tomotopy#78]
Also came across an error with this script:

```python
import tomotopy as tp

model = tp.DTModel()
model.summary()
```

```text
Traceback (most recent call last):
  File "repro.py", line 5, in <module>
    model.summary()
  File "/usr/local/lib/python3.8/site-packages/tomotopy/_summary.py", line 309, in summary
    _call_method_bound(mdl, 'basic_info', file=file)
  File "/usr/local/lib/python3.8/site-packages/tomotopy/_summary.py", line 35, in _call_method_bound
    return glob_methods[method + '_' + cname](mdl, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/tomotopy/_summary.py", line 56, in basic_info_LDAModel
    print('| Removed Vocabs: {}'.format(' '.join(mdl.removed_top_words) if mdl.removed_top_words else '<NA>'), file=file)
Exception: src/TopicModel/DTModel.hpp (436): No document with timepoint = 0
```

**Comment by bab2min** [comment]
This is also identified as an error caused by accessing uninitialized data. It will be fixed at the next update too.
Thank you again for your reporting!

**Comment by bab2min** [comment]
fixed at version 0.10.0

**Comment by ankane** [comment]
Thanks @bab2min!
